### DISCUSSION TITLE
Issue with templates

### DISCUSSION THREAD

**Discussion by naimrathod** [projectdiscovery/nuclei-templates#7638] [Q&A] [answered]
when i run nuclei without templates its work fine but when i add any templates 
its show  

```
nuclei -u google.com -t CVE-2023-24488.yaml                                        

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.8

                projectdiscovery.io

[INF] Current nuclei version: v2.9.8 (latest)
[INF] Current nuclei-templates version: v9.5.4 (latest)
[INF] New templates added in latest release: 51
[INF] Templates loaded for current scan: 1
[INF] Targets loaded for current scan: 1
[INF] Running httpx on input host
[INF] Found 1 URL from httpx
[INF] No results found. Better luck next time!
```

templates not work for me

**Comment by MetzinAround** [comment]
Looks like you're running only 1 template against google.com and it's not finding any results, which is likely expected.

**Comment by naimrathod** [comment]
google.com is the sample any vulnarable website like vulnweb or any other when i add any   template not working

**Comment by ehsandeep** [comment]
@Mrx71 you can run the template either by ID or FILE PATH, in this case, you are running with file path which is invalid.

valid examples:

```
nuclei -t http/cves/2023/CVE-2023-24488.yaml -target google.com
nuclei -id CVE-2023-24488 -target google.com
```

if you are not sure about file path, make sure to use id.

**Reply by naimrathod** [reply]
not working either 

└─# nuclei -id CVE-2023-24488 -target http://testphp.vulnweb.com/

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.8

                projectdiscovery.io

[INF] Current nuclei version: v2.9.8 (latest)
[INF] Current nuclei-templates version: v9.5.4 (latest)
[INF] New templates added in latest release: 51
[INF] Templates loaded for current scan: 1
[INF] Targets loaded for current scan: 1
[INF] No results found. Better luck next time!

**Reply by ehsandeep** [reply, marked as answer]
@Mrx71 because `http://testphp.vulnweb.com/` is not vulnerable for CVE-2023-24488, it's working as expected, scan got completed, and no results are found, you will only see results when there are any.

**Reply by naimrathod** [reply]
which means all templates work like that ??

**Comment by tomtom8887-tech** [comment]
It always appears better luck time, does nuclei need a large bandwidth. to be smooth?
